typescript/cli/src/deploy/warp.ts

```diff
@@ -197,15 +202,37 @@ async function executeDeploy(
     multiProvider,
     contractVerifier,
   );
+  const modifiedConfig: WarpRouteDeployConfig = await promiseObjAll(
```

Comment:
Good call out. didn't run into during my tests, but i will investigate more

Reply:
I don't think this is an issue because the Ism will be deployed first, and then the Hook since we're `await`ing. im assuming each deployment call will increment the nonce sequentially. Also tested with

```yaml
beta:
  interchainSecurityModule:
    modules:
      - relayer: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
        type: trustedRelayerIsm
      - domains: {}
        owner: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
        type: defaultFallbackRoutingIsm
    threshold: 1
    type: staticAggregationIsm
  isNft: false
  mailbox: "0xDde063eBe8E85D666AD99f731B4Dbf8C98F29708"
  owner: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
  type: native
  hook:
    hooks:
      - address: "0x48e6c30B97748d1e2e03bf3e9FbE3890ca5f8CCA"
        type: merkleTreeHook
      - address: "0x1c32f8818e38a50d37d1E98c72B9516a50985227"
        beneficiary: "0xA5199B345C805010b0A9A2BC489Df6100De53830"
        oracleConfig:
          stridetestnet:
            gasPrice: "10000"
            tokenExchangeRate: "1000"
        oracleKey: "0xA5199B345C805010b0A9A2BC489Df6100De53830"
        overhead:
          stridetestnet: 100000
        owner: "0xA5199B345C805010b0A9A2BC489Df6100De53830"
        type: interchainGasPaymaster
    type: aggregationHook
gamma:
  interchainSecurityModule:
    modules:
      - relayer: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
        type: trustedRelayerIsm
      - domains: {}
        owner: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
        type: defaultFallbackRoutingIsm
    threshold: 1
    type: staticAggregationIsm
  isNft: false
  mailbox: "0x56fC17a65ccFEC6B7ad0aDe9BD9416CB365B9BE8"
  owner: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
  type: native
  hook:
    hooks:
      - address: "0x48e6c30B97748d1e2e03bf3e9FbE3890ca5f8CCA"
        type: merkleTreeHook
      - address: "0x1c32f8818e38a50d37d1E98c72B9516a50985227"
        beneficiary: "0xA5199B345C805010b0A9A2BC489Df6100De53830"
        oracleConfig:
          stridetestnet:
            gasPrice: "10000"
            tokenExchangeRate: "1000"
        oracleKey: "0xA5199B345C805010b0A9A2BC489Df6100De53830"
        overhead:
          stridetestnet: 100000
        owner: "0xA5199B345C805010b0A9A2BC489Df6100De53830"
        type: interchainGasPaymaster
    type: aggregationHook


```